README.md

```diff
@@ -0,0 +1,8 @@
+# ROS 2 Launch Framework
+## Tools for launching multiple processes and for writing tests involving multiple processes.
+
+ * [License](LICENSE)
+ * [Contribution Guide](CONTRIBUTING.md)
+ * [Conceptual Documentation](https://docs.ros.org/en/rolling/Concepts/Basic/About-Launch.html)
+ * [Tutorials](https://docs.ros.org/en/rolling/Tutorials/Intermediate/Launch/Launch-Main.html)
+ * [Sphinx Documentation](https://docs.ros.org/en/rolling/p/launch/architecture.html)
```

Comment:
What about just calling it "Package Documentation"?

Also, we might want to link to the docs root instead of a specific page: https://docs.ros.org/en/rolling/p/launch/